Gopkg.toml

```diff
@@ -6,6 +6,10 @@
   name = "github.com/spf13/cobra"
   version = "0.0.2"
 
+[[override]]
+  name = "github.com/prometheus/client_golang"
```

Comment:
This package isn't really used in the SDK itself right now so we can remove this. We'll add it back in later once we actually use this to record default metrics inside the SDK.
For now we can add it to the Gopkg.toml template of the operator:
https://github.com/operator-framework/operator-sdk/blob/master/pkg/generator/templates.go#L372